### PR TITLE
[ES/FR DateTimeV2] Fixed "ago" not recognized correctly (#2478)

### PR DESCRIPTION
**Pull request by LionbridgeCSII**
Fix to issue #2478.

Test cases added to French and Spanish DateTimeModels.

Also replaced undefined regexes ^[.] with ^\b$ in French yaml file because ^[.] was matching in certain cases and giving issues (for example, here "30 minutes." was extracted as "30 minutes in the future" because FutureSuffixRegex was undefined but matched in the afterStr "." of the duration extraction "30 minutes").

### CODE REVIEW

File: Specs/DateTime/French/DateTimeModel.json [hunk 1]
```diff
@@ -20902,5 +20902,53 @@
...
+              "type": "datetime",
+              "value": "2021-05-20 15:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Je l'ai commandé il y a 24 heures.",
```
Comment (tellarin) [inline Specs/DateTime/French/DateTimeModel.json:20931]: Can "il ya" appear in informal French? If so, we should support it if it doesn't cause undesired side effects.
Should we also have more specs with depuis?
And lastly, can you this spec? "Il y a à peu près un mois".
If not supported and too complex to address, add this new spec as disabled and open an issue to fix it. Thanks.

**Review by tellarin — CHANGES_REQUESTED** [review]

File: Specs/DateTime/French/DateTimeModel.json (hunk 1 above)
Comment (LionbridgeCSII) [reply Specs/DateTime/French/DateTimeModel.json:20931]: test cases added.
"Il y a à peu près un mois" is now parsed as date but the modifier "à peu près" is ignored. 
The problem is that Merged looks for modifiers only at the beginning/end of the extraction, so a larger modification is required to target modifiers within the extraction.

Comment (tellarin) [reply Specs/DateTime/French/DateTimeModel.json:20931]: Got it. Let's open an issue for it, but this is lower priority for now.

**Review by tellarin — COMMENTED** [review]
Approved.

**Pull request merged**